### ISSUE TITLE
[Bug]: The screen reader is not reading the contents in the gird after ShimmeredDetailsList goes from Shimmer to displaying contents.

### ISSUE THREAD

**Issue by brandonthomas** [microsoft/fluentui#27455]
### Library

React / v8 (@fluentui/react)

### System Info

```shell
System:
    OS: Windows 10 10.0.22621
    CPU: (12) x64 Intel(R) Core(TM) i7-8700K CPU @ 3.70GHz
    Memory: 10.50 GB / 31.94 GB
  Browsers:
    Edge: Spartan (44.22621.1413.0), Chromium (111.0.1661.41)
    Internet Explorer: 11.0.22621.1
```

### Are you reporting Accessibility issue?

yes

### Reproduction

https://developer.microsoft.com/en-us/fluentui#/controls/web/detailslist/shimmer

## Actual Behavior
When using screen reader shortcut `t` to select the next table, screen reader users are unable to access the contents of the list past the headers. 

## Expected Behavior
After loading (and aria-busy is set to false), users can reach contents of list through virtual cursor.

After doing some investigating and speaking to @smhigley it looks like this might be an AT bug or maybe* UIA? Something is getting confused and the AT ignores the DOM even though aria-busy is set to false. If we remove `aria-busy` the table works as expected. I'll have A11Y folks follow up with Edge and NVDA and JAWS but for now the proposed fix is that we remove `aria-busy` instead of setting it to false.

Priority is set to high because this has been marked as a Sev1 A11Y bug and it will impact SLA for Fluent and downstreams.

### Requested priority

High

### Are you willing to submit a PR to fix?

yes

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

**Comment by smhigley** [comment]
Debugged this with @brandonthomas, turns out it's a browser issue and it was fixed sometime between Chromium v111 (current) and v113.
